nvidia-ubuntu/Dockerfile

```diff
@@ -5,30 +5,27 @@ LABEL maintainer="David Beckingsale <david@llnl.gov>,vsoch"
 
 ENV cuda_version=$cuda_version
 ENV CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-${cuda_version}
+ENV spack_commit=develop
 
 ENV DEBIAN_FRONTEND=noninteractive
 ENV TZ=America/Los_Angeles
 
-RUN apt-get -qq update && \
-    apt-get -qq install -y --no-install-recommends \
-      wget \
-      python3-dev \
-      python3-pip \
-      build-essential \
-      sudo \
-      git \
-      vim \
-      dh-autoreconf \
-      ninja-build \
-      ca-certificates
+WORKDIR /opt
+COPY ./scripts /opt/scripts
+COPY ./ubuntu/scripts/* /opt/scripts
```

Comment:
Because the scripts in the upper directory aren't ubuntu specific.  The spack setup and cmake fetch are both general operations.  I can move them in if that's your preference, but the spack setup one would make sense for the alpine containers too if they are going to run spack.

Reply:
Ah, one source with subdirectories like you mention below would work, I can do that.

Reply:
How about:

```
./scripts/
    ubuntu/
    generic/
```
so we can just do one copy for everyone?

Reply:
Yeah, definitely.  I just put the general ones at ./scripts for now but that's basically what I'm doing.